bin/audioldm2

```diff
@@ -11,6 +11,15 @@ matplotlib_logger.setLevel(logging.WARNING)
 
 parser = argparse.ArgumentParser()
 
+parser.add_argument(
+    "--mode",
+    type=str,
+    required=False,
+    default="generation",
+    help="generation: text-to-audio generation; transfer: style transfer",
+    choices=["generation", "transfer", "sr_ip"]
```

Comment:
Possibly "super_resolution_inpainting" is a bit clearer than "sr_ip"? But I do agree "sr_ip" is short and easier to use.